src/servers/main.cc

```diff
@@ -82,7 +82,8 @@ int32_t http_port_ = 8000;
 int32_t http_health_port_ = -1;
 std::vector<int32_t> http_ports_;
 std::vector<std::string> endpoint_names = {
-    "status", "health", "infer", "modelcontrol", "sharedmemorycontrol"};
+    "status",    "health", "infer", "modelcontrol", "sharedmemorycontrol",
```

Comment:
Did the formatter add this extra whitespace?

Reply:
Yes